app/utils/SitemapWriter.scala

```diff
@@ -18,7 +18,7 @@ case class SitemapURL(url: String,
 
 class SitemapWriter @Inject()(publicationDAO: PublicationDAO, wkConf: WkConf)(implicit ec: ExecutionContext)
     extends FoxImplicits {
-  private val proxyURLs = wkConf.Proxy.routes.map(SitemapURL(_))
+  private val proxyURLs = wkConf.Proxy.routes.filter(!_.contains("*")).map(SitemapURL(_))
```

Comment:
Note for reviewer: This is unrelated, slipped in here. We noticed that the wildcard feature of the URL proxy conflicts with listing the proxy URLs in the sitemap. If the feature is used, those should not be listed (as with the wildcard they are not real URLs)